frontend/packages/jobs/src/functions/processGenerateReview.ts

```diff
@@ -25,22 +26,54 @@ export const processGenerateReview = async (
   payload: GenerateReviewPayload,
 ): Promise<string> => {
   try {
-    const docs = await prisma.doc.findMany({
+    // Get repository installationId
+    const repository = await prisma.repository.findUnique({
```

Comment:
The @liam-hq/jobs could not call the supabase client because the implementation to create the supabase client is in @liam-hq/app. I will fix later!

https://github.com/liam-hq/liam/blob/c11229605374acb9e32ecda3f511850d6df709f4/frontend/apps/app/libs/db/server.ts#L4